lib/pymedphys/_experimental/streamlit/apps/wlutz/_angles.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (C) 2020 Cancer Care Associates
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import warnings
+
+from pymedphys._imports import numpy as np
+from pymedphys._imports import pandas as pd
+
+from pymedphys._utilities import transforms as pmp_transforms
+
+from . import _utilities
+
+GANTRY_EXPECTED_SPEED_LIMIT = 1  # RPM
+COLLIMATOR_EXPECTED_SPEED_LIMIT = 2.7  # RPM
+NOISE_BUFFER_FACTOR = 5  # To allow a noisy point to not trigger the speed limit
+
+
+def make_icom_angles_continuous(icom_datasets):
+    try:
+        angle_speed_check(icom_datasets)
+    except ValueError:
+        icom_datasets["gantry"] = attempt_to_make_angle_continuous(
+            icom_datasets["datetime"],
+            icom_datasets["gantry"].to_numpy(),
+            GANTRY_EXPECTED_SPEED_LIMIT * NOISE_BUFFER_FACTOR,
+        )
+        icom_datasets["collimator"] = attempt_to_make_angle_continuous(
+            icom_datasets["datetime"],
+            icom_datasets["collimator"].to_numpy(),
+            COLLIMATOR_EXPECTED_SPEED_LIMIT * NOISE_BUFFER_FACTOR,
+        )
+
+    angle_speed_check(icom_datasets)
+
+    return icom_datasets
+
+
+def fix_bipolar_angle(angle: "pd.Series"):
+    output = angle.to_numpy()
+    output[output < 0] = output[output < 0] + 360
+
+    output = pmp_transforms.convert_IEC_angle_to_bipolar(output)
+
+    return output
+
+
+def determine_speed(angle, time):
+    diff_angle = np.diff(angle) / 360
+    diff_time = pd.Series(np.diff(time)).dt.total_seconds().to_numpy() / 60
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=RuntimeWarning)
+        rpm = diff_angle / diff_time
+
+    return np.abs(rpm)
+
+
+def get_collimator_and_gantry_flags(icom_datasets):
```

Comment:
Yup, I like that convention, hadn't heard it before, but it seems obvious now that you mention it. I'll rename now so that it's not forgotten, will leave the docstring for when prepping for elevation.